infra/bridge/src/controllers/website.js

```diff
@@ -64,7 +64,9 @@ router.post('/verify', websiteVerify, async (req, res) => {
     const attestation = await generateAttestation(
       AttestationTypes.WEBSITE,
       attestationBody,
-      remoteOrigin,
+      {
+        uniqueId: remoteOrigin
```

Comment:
Are there any guarantees this is unique? I guess multiple users could attest to the same URL?

Reply:
No, we don't have a unique check for this. Multiple addresses can attest with the same domain, subdomain or on a different port. We don't have a unique/primary constraint in the database column either, so it should not throw in case of a duplicate domain.

But we can switch this with singed message. That should be unique for each address, even if the domain is same.